ccm_web/client/src/App.tsx

```diff
@@ -1,7 +1,6 @@
 import React, { useEffect, useState } from 'react'
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
+import { Route, Switch, useLocation } from 'react-router-dom'
```

Comment:
I alluded to this in the above note, this is part of changes that the`react-router-dom` is recommending, using their new hooks and `<Route>{children}</Route>` instead of `<Route render={() => {}} />`. See https://reactrouter.com/docs/en/v6/upgrading/v5 and https://reacttraining.com/blog/react-router-v5-1/.